tasks/test.yml

```diff
@@ -137,7 +137,7 @@ tasks:
     desc: Runs PHPCS with Drupal Coding Standards
     summary: |
       Check your code against Drupal's coding standards. To override the default
-      ruleset, copy vendor/lullabot/drainpipe/scaffold/phpcs.xml to phpcs.xml
+      ruleset, copy vendor/lullabot/drainpipe/scaffold/phpcs.xml.dist to phpcs.xml.dist
```

Comment:
```suggestion
      ruleset, copy vendor/lullabot/drainpipe/scaffold/phpcs.xml.dist to phpcs.xml
```

Reply:
I don't understand this suggestion.

Reply:
The instructions are to copy the file to the same name?

Reply:
It is the same pattern that was there before I made the change. All I did was change the file name.

Reply:
I think this is a bit confusing, because `.dist` usually means "shipped with the project" not "local edits". I think `phpcs.xml` would be more canonical... but we should probably also automatically copy `phpcs.xml.dist` to the root of the repo?

Reply:
I'm confused I thought the whole point of the ticket was to have drainpipe generate the projects phpcs.xml.dist instead of the phpcs.xml

Reply:
I guess I'm confused because the instruction says "To override the default ruleset" in the beginning, which to mean assumes this is for a project-specific config file. I would assume the scaffolding file is being copied to the root of the project, which is what I see based on `"[project-root]/phpcs.xml.dist": "scaffold/phpcs.xml.dist"` being in the composer.json...

Reply:
Oh, I see... ya I don't know where my brain was at that day. Maybe that should be changed to direct the user to the readme because I don't see [these instructions](https://github.com/Lullabot/drainpipe/pull/562/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R236) fitting into that.